rdt-open-reader/src/covidIndonesia/resources/sample-collection-form_in.properties

```diff
@@ -63,19 +63,19 @@ sample-collection-form-original.step2.other_sample_type_lbl.text = Jenis Sampel
 sample-collection-form-original.step2.sample_expiration_date.v_required.err = Bagian ini diperlukan
 sample-collection-form-original.step2.doctor_phone_number_lbl.text = Nomor Telepon Dokter
 sample-collection-form-original.step5.qr_code_reader.hint =
-sample-collection-form-original.step2.name_of_primary_care_facility.v_required.err = Bagian ini diperlukan
+sample-collection-form-original.step2.name_of_health_facility.v_required.err = Bagian ini diperlukan
 sample-collection-form-original.step7.lbl_store_specimen_for_courier.text = Simpan sampel dengan tepat sampai diambil oleh kurir
 sample-collection-form-original.step2.vtm_brand.options.other.text = Lainnya
 sample-collection-form-original.step6.title = Koleksi sampel
-sample-collection-form-original.step2.name_of_primary_care_facility.options.dinas_kesehatan.text = Dinas Kesehatan
+sample-collection-form-original.step2.name_of_health_facility.options.dinas_kesehatan.text = Dinas Kesehatan
 sample-collection-form-original.step2.patient_health_status.label = Status kesehatan pasien?
 sample-collection-form-original.step2.sample_type.v_required.err = Bagian ini diperlukan
 sample-collection-form-original.step2.title = Koleksi sampel
 sample-collection-form-original.step3.next_label = TERUS
 sample-collection-form-original.step2.gsi_sample_number_lbl.text = Nomor Spesimen GSI
 sample-collection-form-original.step2.health_facility_type.options.clinic.text = Klinik
 sample-collection-form-original.step4.lbl_scan_respiratory_specimen_barcode.text = Pindai kode batang sampel
-sample-collection-form-original.step2.name_of_primary_care_facility.label = Nama Fasyankes
+sample-collection-form-original.step2.name_of_healthfacility.label = Nama Fasyankes
```

Comment:
`name_of_healthfacility` looks wrong?

Reply:
Yeah, I've done this so that it's handled by `Transifex`